web/satellite/src/components/team/TeamMemberItem.vue

```diff
@@ -4,7 +4,7 @@
 <template>
     <div class="user-container">
         <div class="user-container__avatar">
-            <h1>m</h1>
+            <h1>{{projectMember.user.firstName.slice(0,1)}}</h1>
```

Comment:
It should be computed value for now

Reply:
Why computed required? ProjectMembers name will not change for this item